public/app/features/live/LiveConnectionWarning.tsx

```diff
@@ -44,24 +44,36 @@ export class LiveConnectionCorner extends PureComponent<Props, State> {
   render() {
     const { show } = this.state;
     if (show) {
-      return <div className={this.styles.corner} title="The server is disconnected" />;
+      if (!(window as any).grafanaBootData?.user?.isSignedIn) {
```

Comment:
Might want to use contextSrv.isSignedIn? 
https://github.com/grafana/grafana/blob/0d044285a9c646d4317f9f9da8694b70e7f9fc73/public/app/core/components/sidemenu/BottomSection.tsx#L12